decidim-admin/app/commands/decidim/admin/bulk_action.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class BulkAction < Decidim::Command
+      # Public: Initializes the command.
+
+      def initialize(user, action, selected_moderations)
+        @user = user
```

Comment:
+ all needed changes + Method signature ( what every param is referring to )
```suggestion
      def initialize(current_user, action, selected_moderations)
        @current_user = current_user
```

Reply:
Done